source/reference/template-apps.txt

```diff
@@ -32,41 +27,50 @@ an app based on a template using any of the following interfaces:
 Create a Template App
 ---------------------
 
-You can create a template app using the UI on the Atlas App Services site, 
-the Realm CLI, or Admin API. Choose the tab for whichever 
+You can create a template app using the UI on the Atlas App Services
+site, the Realm CLI, or the Admin API. Choose the tab for whichever
 option is most convenient for you.
 
-.. tabs::
+.. tabs-realm-admin-interfaces::
    
-   .. tab:: Atlas App Services UI
+   .. tab::
       :tabid: ui
-      
-      Follow the procedure described in the 
-      :ref:`create-an-app` guide, and select
-      the :guilabel:`Create App from Template` tab in the :guilabel:`Create a new App` step. 
 
-   .. tab:: {+cli+}
+      You can create a template App using the same command as you would
+      use to create a blank App. To learn how to create an App Services
+      App, see :ref:`Create an App <create-app>` and follow the
+      :guilabel:`Template App` instructions.
+
+   .. tab::
       :tabid: cli
 
-      Follow the procedure described in the 
-      :ref:`Create an App with {+cli+} <create-app-cli>` guide and include
-      the ``template`` flag followed by one of the template app IDs: 
+      You can create a template App using the same command as you would
+      use to create a blank App. To learn how to create an App Services
+      App, see :ref:`Create an App <create-app>`.
```

Comment:
this tab,  and the Admin API tab, do not include the bit added to the UI tab:
 `and follow the  :guilabel:`Template App` instructions.`